examples/large_models/deepspeed/opt/Readme.md

```diff
@@ -39,56 +45,3 @@ torchserve --start --ncs --model-store model_store --models opt.tar.gz
 curl -v "http://localhost:8080/predictions/opt" -T sample_text.txt
 ```
 
-## Option 2: Using model name
-
-### Step 1: Update initialize in custom_handler.py
-```python
-    def initialize(self, ctx: Context):
-        """In this initialize function, the HF large model is loaded and
-        partitioned using DeepSpeed.
-        Args:
-            ctx (context): It is a JSON Object containing information
-            pertaining to the model artefacts parameters.
-        """
-        super().initialize(ctx)
-        model_dir = ctx.system_properties.get("model_dir")
-        self.max_length = int(ctx.model_yaml_config["handler"]["max_length"])
-        self.max_new_tokens = int(ctx.model_yaml_config["handler"]["max_new_tokens"])
-        seed = int(ctx.model_yaml_config["handler"]["manual_seed"])
-        torch.manual_seed(seed)
-
-        self.tokenizer = AutoTokenizer.from_pretrained(model_dir, padding_side="left")
-        self.tokenizer.pad_token = self.tokenizer.eos_token
-        self.model = AutoModelForCausalLM.from_pretrained(model_dir, torch_dtype=torch.float16)
-        self.model.eval()
-
-        ds_engine = get_ds_engine(self.model, ctx)
```

Comment:
Suggest keep this portion only and mention that in your custom handler add these 3-4 lines

Reply:
Its not clear what Option 2 is doing and if its relevant with the updated code ?